jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -84,6 +84,7 @@
         <programRunner implementation="software.aws.toolkits.jetbrains.services.lambda.execution.remote.RemoteLambdaRunner"/>
         <fileBasedIndex implementation="software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerIndex"/>
         <codeInsight.lineMarkerProvider language="" implementationClass="software.aws.toolkits.jetbrains.services.lambda.upload.LambdaLineMarker"/>
+        <runLineMarkerContributor language="" implementationClass="software.aws.toolkits.jetbrains.services.lambda.execution.template.LambdaRunLineMarkerContributor"/>
```

Comment:
Language should be YAML

Reply:
I'd like to use this class to support JSON as well later.

Reply:
we can register it with both languages when we get there